.github/workflows/ci.yml

```diff
@@ -0,0 +1,74 @@
+name: Continuous integration
+
+on:
+  workflow_dispatch: ~
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
+    branches:
+      - develop
+      - main
+
+jobs:
+  ci:
+    name: Linting, tests and coverage
+    runs-on: ubuntu-22.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php:
+          - version: 5.6
+            unittest-version: Legacy
+          - version: 7.0
+            unittest-version: PHP7_0
+          - version: 7.1
+            unittest-version: PHP7_0
+          - version: 7.2
+            unittest-version: PHP7_2
+          - version: 7.3
+            unittest-version: PHP7_2
+          - version: 7.4
+            unittest-version: PHP7_2
+          - version: 8.0
+            unittest-version: PHP7_2
+          - version: 8.1
+            unittest-version: PHP8_1
+          - version: 8.2
+            unittest-version: PHP8_1
+          - version: 8.3
+            unittest-version: PHP8_1
```

Comment:
Shouldn't this be

```suggestion
          - version: 5.6
            unittest-version: Legacy
          - version: 7.0
            unittest-version: PHP7_0
          - version: 7.1
            unittest-version: PHP7_1
          - version: 7.2
            unittest-version: PHP7_2
          - version: 7.3
            unittest-version: PHP7_3
          - version: 7.4
            unittest-version: PHP7_4
          - version: 8.0
            unittest-version: PHP8_0
          - version: 8.1
            unittest-version: PHP8_1
          - version: 8.2
            unittest-version: PHP8_2
          - version: 8.3
            unittest-version: PHP8_3
```

Reply:
No, we got a few folders for different breaking changes in some PHP versions: https://github.com/alma/alma-php-client/tree/main/tests/Unit